specs/electra/beacon-chain.md

```diff
@@ -305,67 +292,32 @@ class BeaconBlockBody(Container):
     voluntary_exits: List[SignedVoluntaryExit, MAX_VOLUNTARY_EXITS]
     sync_aggregate: SyncAggregate
     # Execution
-    execution_payload: ExecutionPayload  # [Modified in Electra:EIP6110:EIP7002]
+    execution_payload: ExecutionPayload
     bls_to_execution_changes: List[SignedBLSToExecutionChange, MAX_BLS_TO_EXECUTION_CHANGES]
     blob_kzg_commitments: List[KZGCommitment, MAX_BLOB_COMMITMENTS_PER_BLOCK]
+    requests: ExecutionLayerRequests  # [New in Electra]
```

Comment:
A thought on making the field name more specific:
```suggestion
    execution_requests: ExecutionLayerRequests  # [New in Electra]
```

Reply:
Ok, will update :)